api_v2/src/esl-change-shift/esl-change-shift.repository.ts

```diff
@@ -95,4 +95,32 @@ export class EslChangeShiftRepository {
   async runTransaction<T>(actions: Prisma.PrismaPromise<T>[]): Promise<T[]> {
     return this.prisma.$transaction(actions);
   }
+
+  async getUserOffsetCredits(userId: number) {
+    return this.prisma.offsetRequest.findMany({
+      where: {
+        userId,
+        isLeaderApproved: true,
+        isUsed: false,
+      },
+      select: {
+        creditEarned: true,
+      },
+    });
+  }
```

Comment:
```suggestion
  async getUserOffsetCredits(userId: number) {
    return this.prisma.offsetRequest.findMany({
      where: {
        userId,
        isLeaderApproved: true,
        isUsed: false,
      },
      select: {
        creditEarned: true,
      },
    });
  }
```

this should be in offset repository